Dockerfile

```diff
@@ -1,18 +1,23 @@
-# ---- Base image ----
-FROM hmctspublic.azurecr.io/base/node:20-alpine as base
+FROM hmctspublic.azurecr.io/base/node:20-alpine as build
+
 USER root
 RUN corepack enable
-COPY --chown=hmcts:hmcts . .
 USER hmcts
 
-# ---- Build image ----
+WORKDIR /opt/app
+
+COPY --chown=hmcts:hmcts . .
+
+RUN PUPPETEER_SKIP_DOWNLOAD=true yarn install
```

Comment:
should really be a second stage that only installs production dependencies as well

Reply:
Aye, a wee yarn workspaces focus --production wouldn't go amiss. That's next on the agenda (different PR)